app/src/main/java/io/homeassistant/companion/android/widgets/StaticWidget.kt

```diff
@@ -172,10 +176,11 @@ class StaticWidget : AppWidgetProvider() {
             staticWidgetDao.add(StaticWidgetEntity(
                 appWidgetId,
                 entitySelection,
-                attributeSelection,
+                attributeSelection?.joinToString(","),
                 labelSelection,
                 textSizeSelection?.toFloatOrNull() ?: 30F,
-                if (separatorSelection == "") " " else separatorSelection ?: " "
+                stateSeparatorSelection ?: "",
+                attributeSeparatorSelection ?: ""
```

Comment:
Correct. All of the previous widgets will migrate to have a space as a separator, however, new ones will have nothing, if user chooses "No separator" and doesn't type anything in the field. I felt this was a better approach as it allows to, for example, add `unit_of_measurement` right after the value like this "1.24‎€", previously it would have been like this "1.24 ‎€". Besides users still have an option to add space in the separator field.